.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: User Feedback
+assignees: ''
+
+---
+
+## Description
+A clear and concise description of what the bug is.
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Actual behavior
+A clear and concise description of what you have experienced instead.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## Configuration
+ - Device: [e.g. iPhone8]
+ - OS: [e.g. iOS11.2]
+ - Frameworks versions (leave the ones you're using):
+   - MapboxVisionNative [e.g. 0.3.0]
+   - MapboxVisionARNative [e.g. 0.3.0]
+   - MapboxVisionSafetyNative [e.g. 0.3.0]
+   - MapboxVision [e.g. 0.3.0]
+   - MapboxVisionAR [e.g. 0.3.0]
+   - MapboxVisionSafetyNative [e.g. 0.3.0]
+
```

Comment:
Probably we can ask to paste state from `Cartfile.resolved` file? What do you think?

Reply:
We care only about libraries we support. I consider full `Cartfile.resolved` or `Podfile` in case of CocoaPods unnecessary information.